modules/cloud/pages/couchbase-containers-faq.adoc

```diff
@@ -0,0 +1,32 @@
+= Frequently Asked Questions about Couchbase Containers
+:description: Frequently asked questions about Couchbase Containers 
+
+[abstract]
+{description}
+
+== Do we support modifying the official Couchbase container?
```

Comment:
"Do we support"... reads a bit strange as a customer, who is the we ?  me the customer ?  
IMHO,  "Do Couchbase support"... reads better.